core/domain/question_fetchers.py

```diff
@@ -63,14 +79,21 @@ def get_questions_and_skill_descriptions_by_skill_ids(
 
     for skill_ids_list in grouped_skill_ids:
         skills = skill_models.SkillModel.get_multi(skill_ids_list)
-        grouped_skill_descriptions.append(
-            [skill.description if skill else None for skill in skills])
+        skill_descriptions: List[Optional[str]] = []
+        for skill in skills:
+            if skill:
+                # Ruling out the possibility of None for mypy type checking.
```

Comment:
```suggestion
```

My bad, since its asserting that its a `str`, this isn't needed

Reply:
Done